tests/pages/services/TasksTable-cy.js

```diff
@@ -87,4 +87,47 @@ describe("Tasks Table", function() {
       });
     });
   });
+
+  context("Service tasks checkbox", function() {
+    beforeEach(function() {
+      cy.configureCluster({
+        mesos: "1-service-with-executor-task"
+      });
+      cy.visitUrl({ url: "/services/detail/%2Fcassandra/tasks?_k=rh67gf" });
+      cy.get("table tr").find(".form-element-checkbox").as("checkboxes");
+    });
+
+    function assertCheckboxLength() {
+      cy.get("@checkboxes").should("have.length", 2);
+    }
+
+    it("Select all tasks available", function() {
```

Comment:
@ahoskins having a wait is somewhat arbitrary and brittle. So long as we have the `uniqueProperty` set to "id" we will be fine. Good to see you're thinking of good test cases 👍